src/observers/OmnisharpStatusBarObserver.ts

```diff
@@ -42,6 +42,9 @@ export class OmnisharpStatusBarObserver extends BaseStatusBarItemObserver {
             case EventType.InstallationStart:
                 this.SetAndShowStatusBar("$(desktop-download) Installing packages...", '', '', `Installing package '${(<InstallationStart>event).packageDescription}'`);
                 break;
+            case EventType.InstallationSuccess:
+                this.ResetAndHideStatusBar();
```

Comment:
Could you add a test for this in OmnisharpStatusBarObserver.test.ts?

Something like:
```
    test('InstallationSuccess: Status bar is hidden and the attributes are set to undefined', () => {
        let installationEvent = new InstallationStart("somePackage");
        observer.post(installationEvent);

        let successEvent = new InstallationSuccess();
        observer.post(successEvent);

        expect(hideCalled).to.be.true;
        expect(statusBarItem.text).to.be.undefined;
        expect(statusBarItem.command).to.be.undefined;
        expect(statusBarItem.color).to.be.undefined;
    });
```